templates/development.php

```diff
@@ -249,13 +249,47 @@ class="copy-clipboard"
         };
 
         for (var i = 0, imax = callStackItems.length; i < imax; ++i) {
-            refreshCallStackItemCode(callStackItems[i]);
+            let stackItem = callStackItems[i];
+            refreshCallStackItemCode(stackItem);
 
             // toggle code block visibility
-            callStackItems[i].getElementsByClassName('element-wrap')[0].addEventListener('click', function (event) {
+            stackItem.querySelector('.show-arguments-toggle')?.addEventListener('click', function (e) {
+                e.stopPropagation()
```

Comment:
Either let's add `;` everywhere or remove it everywhere. I'm for adding it.

Reply:
I'll add it in the last PR in the sequence